draft-add-requirements.md

```diff
@@ -142,18 +243,100 @@ Narrative required.
 
 ### Private enterprise names
 
-Narrative required.
+As stated above, an enterprise-managed Wi-Fi network might be able to access both the Internet and a private
+intranet. The private domains managed by the enterprise might only be resolvable using a specific DNS resolver,
+hence use of that resolver is essential for such domains. However it does not necessarily mean that all queries
+for all domains have to flow through that resolver.
+
+Only sending the necessary queries through the enterprise resolver, and not generic Internet queries, has the privacy
+benefit of only exposing traffic to the enterprise that fall within a limited set of domains.
+
+Using encrypted DNS for private names also opens up the possibility of doing private name resolution outside
+of the content of a VPN or managed network. If the DNS resolver authenticates clients, it can offer its resolver
+for private names on a publicly accessible server, while still limiting the visibility of the DNS traffic.
```

Comment:
This use case is specific to BYOD (personal devices without MDM).

Reply:
Are you saying that enterprise-managed devices will never need to look up private names on a publicly accessible server because they will always have a VPN running before doing such a lookup?

Reply:
No, I don't think that would be right.

I don't think this case is specific to non-MDM devices. Even with MDM devices, devices aren't often configured to know which domains are privately accessible on an enterprise Wi-Fi network. Certainly MDMs could be updated going forward (which wouldn't be possible for pure BYOD cases), but that's not the status quo. Also, if we have a mechanism that works for the pure BYOD case, it wouldn't be necessary to build an MDM-specific solution.

Reply:
> Are you saying that enterprise-managed devices will never need to look up private names on a publicly accessible server because they will always have a VPN running before doing such a lookup?

The configuration on the enterprise-managed devices is governed by the IT admin and users typically no control to change the config. You may want to look into https://tools.ietf.org/html/draft-reddy-add-enterprise-00#section-3 for more details.

Reply:
> 
> 
> No, I don't think that would be right.
> 
> I don't think this case is specific to non-MDM devices. Even with MDM devices, devices aren't often configured to know which domains are privately accessible on an enterprise Wi-Fi network. Certainly MDMs could be updated going forward (which wouldn't be possible for pure BYOD cases), but that's not the status quo. Also, if we have a mechanism that works for the pure BYOD case, it wouldn't be necessary to build an MDM-specific solution.

Agreed, we can say the use case is specific to "BYOD" without explicitly saying "BYOD without MDM". On a side note, MDM is a secure way of configuring endpoints and a secure mechanism will be required for pure BYOD case to be applicable to MDM. Configuration profile could be another way to securely provision the BYOD without MDM (see https://tools.ietf.org/html/draft-reddy-add-enterprise-00#section-5).